packages/block-library/src/post-comments-form/style.scss

```diff
@@ -0,0 +1,24 @@
+$blocks-button__height: 3.1em;
+
+// Styles copied from button block styles.
+.wp-block-post-comments-form input[type="submit"] {
+	color: $white;
+	background-color: #32373c;
```

Comment:
Is there any variable that matches this color?

Reply:
Nope! It's the exact same CSS used in the block button... I grep'd the repo and there's no variable defined for that color (css-var or scss-var)